cypress/integration/login.js

```diff
@@ -20,5 +20,9 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 1)
+
+    })
+    it('Should show current date', function (){
+        cy.get('.calendar--selected').contains('13')
```

Comment:
will it work tomorrow? Think about how you can make it work everyday.

Reply:
Request you to review again. :)